client/components/ScopeDropdown/ScopeDropdown.tsx

```diff
@@ -231,7 +242,9 @@ const ScopeDropdown = (props: Props) => {
 												(scope.slugs && scope.slugs.pageSlug) || '/'
 											}`}
 										>
-											<Icon icon="globe" iconSize={12} />
+											<Tooltip content={`${scope.title} landing page`}>
```

Comment:
Pub titles can get _really_ long and probably aren't suitable for a tooltip. I'm not sure what this one should say. Maybe just "Visit"? I would defer to @djagdish.

Reply:
"Home" maybe ???